framework/src/executioners/EigenExecutionerBase.C

```diff
@@ -319,7 +319,7 @@ EigenExecutionerBase::inversePowerIteration(unsigned int min_iter,
 
     if (echo)
       _console << " ________________________________________________________________________________ "
-               << std::endl;
+               << std::endl << std::flush;
```

Comment:
Yes, possibly. I can give you a fancy input file which has five multiapps a little later. But the other one in `printEigenvalue` seems necessary.